### ISSUE TITLE
How to show the latest block balance of the address?

### ISSUE THREAD

**Issue by rogergangawork** [blockscout/blockscout#3964]
Currently the explorer shows the balance where the last transaction block was seen.

How do I change the last updated block to latest? Where can I change it

**Comment by vbaranov** [comment]
@rogerganga 

>Currently the explorer shows the balance where the last transaction block was seen.

Not really. Blockscout shows the balance from the last block where the balance for this address was changed. I cannot imagine the rationale for the changes, that you proposed. Could you please elaborate on that?

**Comment by rogerganga** [comment]
Hi @vbaranov 

Blockscout has issue when it comes to verifying internal transactions for previous versions of Ethereum.

Some of the internal transactions takes hours/days to change it completed status so many transactions appear as **Awaiting status for internal transactions**.

So if any transfer operations happens internally, the balance does not get reflected completely until the transaction status changes from **Awaiting status for internal transactions** to **Success**.

I was wondering if there is a way to show the latest balance of all account addresses rather than relying on internal transactions to get done.
